PaTTAN Room Displays/PaTTAN Room Displays/NoMeetingsPage.xaml.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Xaml;
+
+namespace PaTTAN_Room_Displays
+{
+	public partial class NoMeetingsPage : ContentPage
+	{
+		public String Message { get; set; }
+
+		public NoMeetingsPage()
+		{
+			// order of this message is importnat must be before comp init.
+			BindingContext = this;
```

Comment:
Why is this bound data, when it's just static text? It could be a literal value in the XAML, unless it might be dynamic.

Reply:
I was trying to work with Binding.  In the end I came to the same conclusion.  Later PR's fix this but setting the text in the XAML